.gitignore

```diff
@@ -6,6 +6,7 @@
 
 # Ignore bundler config.
 /.bundle
+/vendor
```

Comment:
@geta6 If ignore this directory, delete `/vendor/assets/*/.keep` files to remove empty directories.